src/RequestTrait.php

```
@@ -52,4 +84,13 @@ protected function doRequest(Request $request, bool $terminate = false): Respons
 
         return $response;
     }
+
+    private function getInputValue(string $name): string
+    {
+        $input = $this->cssSelect(sprintf('input[name="%s"]', $name));
+        if (count($input) === 0) {
+            return '';
```

Should we throw an exception instead of returning empty here? If we get here, doesn't it mean the developer has messed up and is trying to get a value that's not in the form?

---

Yeah that makes sense

---

@joachim-n care to re-review? Added a `self::assertNotCount(0)`, since there may be multiple `name="op"`

---

Looks good!

---

Awesome, merging :)